### ISSUE TITLE
tickvals and ticktext don't work with api_create

### ISSUE THREAD

**Issue by bcdunbar** [ropensci/plotly#1106]
first referenced in the community forum here https://community.plot.ly/t/tickvals-not-saving-with-api-create/5293 

---

tickvals and ticktext work when plotting offline but not when uploaded via api

```r
p <- plot_ly(
    x=c(1,2,3,4,5,6),
    y=runif(6,0,10),
    type='scatter',
    mode='markers') %>%
  layout(
    xaxis = list(
      tickmode='array',
      tickvals=c(1,3.5,6),
      ticktext=c("start","middle","finish")
    )
  )
p
api_create(p)
```

results in https://plot.ly/~bdun9/2005

**Comment by cpsievert** [comment]
Waiting on https://github.com/plotly/streambed/issues/10654 (I can't fix the problem from the R package itself)

Also, closing as this is duplicate of #1197
